src/mono/mono/metadata/appdomain.c

```diff
@@ -2832,7 +2832,7 @@ mono_alc_load_raw_bytes (MonoAssemblyLoadContext *alc, guint8 *assembly_data, gu
 {
 	MonoAssembly *ass = NULL;
 	MonoImageOpenStatus status;
-	MonoImage *image = mono_image_open_from_data_internal (alc, (char*)assembly_data, raw_assembly_len, FALSE, NULL, refonly, FALSE, NULL, NULL);
+	MonoImage *image = mono_image_open_from_data_internal (alc, (char*)assembly_data, raw_assembly_len, TRUE, NULL, refonly, FALSE, NULL, NULL);
```

Comment:
not a big deal, but for non-`ENABLE_NETCORE`, this can be `FALSE` - `ves_icall_System_AppDomain_LoadAssemblyRaw` does its own memcpy.

(Ideally I'd like to see `ves_icall_System_AppDomain_LoadAssemblyRaw` not do its own copying - i'm pretty sure it leaks those bytes on assembly close. I think the managed array pinning it's doing is not necessary anymore under our newer conservative scanning rules for native code)

Reply:
Oh yeah, I can follow up with a PR to fix up non-netcore. I think I want to keep the backport to a one-liner.